receiver/zookeeperreceiver/metadata.yaml

```diff
@@ -5,25 +5,34 @@ attributes:
     description: State of the Zookeeper server (leader, standalone or follower).
   zk.version:
     description: Zookeeper version of the instance.
+  follower.state:
```

Comment:
I would just call this `state`. 

`server.state` is a a special case because it is a resource attribute, so it needs to be appropriate in a global context. 

This is a data point attribute though, so the context is very clear already.